request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -482,3 +485,12 @@ class RegistrantAdmin(admin.ModelAdmin):
 
 class RegistrarAdmin(admin.ModelAdmin):
     model = Registrar
+
+
+class TimeFlagAdmin(admin.ModelAdmin):
+    model = TimeFlag
+
+    def has_add_permission(self, request):
+        if self.model.objects.count() >= MAX_OBJECTS:
```

Comment:
This should be False for any user as we do not want to add more entries?

Reply:
Maybe we will need a seed script?

Reply:
> Maybe we will need a seed script?

up for discussion with Robert/Jim/Max I guess.

Reply:
> This should be False for any user as we do not want to add more entries?

Done